### DISCUSSION TITLE
인원이 많은방에서 오류

### DISCUSSION THREAD

**Discussion by veplay** [storycraft/node-kakao#635] [Q&A]
인원이 적은방에서는
 let userInfo = chat.Channel.getUserInfo(chat.Sender);

에서
userInfo.Id
userInfo.Nickname

이 두개가 전부 통하는데

인원이 1000명이 넘는방에서는 두개다 null 처리가 되네요.. 이거 오류 어떻게 해결해야할까요?? 

도와주세용.. v3 사용중입니다

**Comment by benesiaxd** [comment]
v3을 그만..

**Reply by veplay** [reply]
v3 의 문제인가요?
